pyproject.toml

```diff
@@ -0,0 +1,111 @@
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[project]
+name = "cloudalerts"
+version = "1.0.0"
+description = "Python tool for accessing mozilla cloud alerting services."
+authors = [
+    "Adam Frank <afrank@mozilla.com>",
+    "Bryan Sieber <bsieber@mozilla.com>",
+    "Chris Valaas <cvalaas@mozilla.com>",
+    "Jorge Spiropulo <jspiropulo@mozilla.com>"
+]
+readme = "README.md"
+python = ">=3.7"
+homepage = "https://github.com/mozilla-it/cloudalerts"
+repository = "https://github.com/mozilla-it/cloudalerts"
+documentation = "https://mozilla-it.github.io/cloudalerts"
+
+classifiers = [
+    "Topic :: Software Development"
+]
+
+################
+#    Tools     #
+################
+
+[tool.poetry]
+name = "cloudalerts"
+version = "1.0.0"
+description = "Python tool for accessing mozilla cloud alerting services."
+authors = [
+    "Adam Frank <afrank@mozilla.com>",
+    "Bryan Sieber <bsieber@mozilla.com>",
+    "Chris Valaas <cvalaas@mozilla.com>",
+    "Jorge Spiropulo <jspiropulo@mozilla.com>"
+]
+
+[tool.poetry.dependencies]
+poetry = "*"
```

Comment:
I don't think this should be here. Maybe in dev deps if at all?

Reply:
I can try and remove this, I can't recall but there was oddness as pip install . will read all these dependencies, so the package handler doesn't need to be known to the dockerfile

Reply:
It does look odd, and likely isn't needed since the "build tools" indicates poetry, will checkback

Reply:
I removed the .packages section, which was apparently messing with the whitelisting and would require poetry as a dependency to use in tox if it was there. 

When removed the ability to use poetry is passed to tox through the whitelisting appropriately

Reply:
Hmm, running "tox" versus docker's running of tox was a bit different.

Had to modify docker to install poetry and stopped installing in tox